metricsx/middleware.go

```diff
@@ -298,16 +318,16 @@ func (sw *Service) UnaryInterceptor(ctx context.Context, req interface{}, info *
 		return resp, err
 	}
 
-	latency := time.Since(start) / time.Millisecond
+	latency := time.Since(start).Milliseconds()
 
 	if err := sw.c.Enqueue(analytics.Page{
-		UserId: sw.o.ClusterID,
-		Properties: analytics.
-			NewProperties().
-			SetPath(info.FullMethod).
-			Set("status", status.Code(err)).
-			Set("latency", latency),
-		Context: sw.context,
+		InstanceId:   sw.instanceId,
+		DeploymentId: sw.o.DeploymentId,
+		Project:      sw.o.Service,
+
+		UrlPath:        info.FullMethod,
+		RequestCode:    int(status.Code(err)),
+		RequestLatency: float64(latency),
```

Comment:
If latency is already an int, then there's no need to convert it to float, as we won't gain any accuracy! Revert to int :)

Reply:
Then I would have to make analytics-go change, PR, and a new version of that first. Yes, I was annoyed that I did not realize this earlier :) Should I do that?